Makefile

```diff
@@ -61,6 +61,8 @@ ifeq (boltdb,$(findstring boltdb,$(COSMOS_BUILD_OPTIONS)))
   BUILD_TAGS += boltdb
 endif
 
+BUILD_TAGS += nativebyteorder
```

Comment:
seems we dont have nativebyteorder in v1.0.x yet

Reply:
it ref the main branch memiavl and store.